resources/git-cheat-sheet.md

```diff
@@ -26,6 +38,12 @@
 
 **`git checkout -b existingBranchName newBranchName`** //create Branch from a Branch
 
+**`git reset HEAD~1`** //takes your branch back one commit, but your changes are still in the working tree (I recommend `git add .` next)
```

Comment:
It should say (I recommend `git add .` next to save the changes in your working tree.)